index.html

```diff
@@ -26,20 +40,20 @@ <h2>The Past</h2>
         </section>
         
         <section class="middle-content">
-        
+
             <h2>Why Did It Have To Be Boxes...</h2>
-            
+
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
```

Comment:
You can also put this in the CSS files by assigning a class name.  Inline is okay, but it is important to know how precedence works on this.  Inline takes precedence over internal (in the head section) and internal takes precedence over external styles.  Sometimes when you can't figure out why something isn't applying it is because of precedence